src/components/AddBook.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+function AddBook() {
+  return (
+    <div>
+      <form>
+        <input type='text' placeholder='Author' />
```

Comment:
- [optional] great job adding the input form. Please note, it gets the author and book title. You added just input for author. Kindly fix that.